### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by petrutlucian94**
We're defining a shared project to be used by rock image tests.

Based on: https://github.com/canonical/cert-manager-rock

### CODE REVIEW

File: .copyright.tmpl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Copyright ${years} ${owner}.
```
Comment (bschimke95) [inline .copyright.tmpl:1]: Is this used for some automation somewhere?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    isort {tox_root}/k8s_test_harness --profile=black --check
+    black {tox_root}/k8s_test_harness --check --diff
+
+[flake8]
+max-line-length = 120
+select = E,W,F,C,N
+# E231 rule is not aware of f-strings
+ignore = W503,E231
+exclude = venv,.git,.tox,.tox_env,.venv,build,dist,*.egg_info
+show-source = true
```
Comment (bschimke95) [inline tox.ini]: nit: missing newline

**Review by bschimke95 — COMMENTED** [review]
Looks good, please add a short README and address the small nits, then we are good to go.

File: .copyright.tmpl (hunk 1 above)
Comment (petrutlucian94) [reply .copyright.tmpl:1]: The tox format job ensures that the files have the license headers, I used the cert-manager-rock project as reference: https://github.com/canonical/cert-manager-rock/blob/655db579265f599516f2ef5df01b43a45ead4f1c/tests/tox.ini#L18

Comment (bschimke95) [reply .copyright.tmpl:1]: sounds good, thanks

**Review by bschimke95 — APPROVED** [review]
LGTM

**Pull request merged**